packages/hashmyfiles.vm/tools/chocolateyinstall.ps1

```diff
@@ -6,9 +6,7 @@ try {
   $category = 'Utilities'
 
   $zipUrl = "https://www.nirsoft.net/utils/hashmyfiles.zip"
-  $zipSha256 = "89db49ec6a3e50f1d76da97ac1289272d1b09b9a330d36f65fdbe1f010f1ae8b"
   $zipUrl_64 = "https://www.nirsoft.net/utils/hashmyfiles-x64.zip"
-  $zipSha256_64 = "be2dc5b9613b72ca44e60b7a1b5332593a868079638ded37cc3ad120e7182b0b"
 
   $executablePath = (VM-Install-From-Zip $toolName $category $zipUrl $zipSha256 $zipUrl_64 $zipSha256_64)[-1]
```

Comment:
Can you remove the unreferenced variables: `$zipSha256` and `$zipSha256_64`?

Reply:
If I do that, it doesn't work anymore 😮

Reply:
Ah, I bet it's because we're using positional arguments and marking them as not mandatory... so we should probably remove the positional component and make them be explicitly set if used. Otherwise, we'll need to pass in null/empty values for the positions. If we do remove the positional component, it may affect other packages as well that would then need to explicitly set them.

Reply:
Idea:

Switch positional arguments 3 & 4 in: https://github.com/mandiant/VM-Packages/blob/3556aab8fd482483c0da4d3f732ca88d7e3cce0e/packages/common.vm/tools/vm.common/vm.common.psm1#L485-L488 then update the packages that use them accordingly. This will allow the hashes to be optional and if used, they don't have to be explicitly declared.

Reply:
@MalwareMechanic 
> Switch positional arguments 3 & 4

But there may be other packages that specify `$zipSha256` and not `zipUrl_64`. I think we need that both arguments are not positional.

Reply:
Hmm you're probably right. We'd need to update the current packages to use the explicit syntax instead of using positional arguments

Reply:
I have done it, check last commit 😉